molecule/default/INSTALL.rst

```diff
@@ -0,0 +1,15 @@
+***********************************
+Delegated driver installation guide
```

Comment:
Is this root molecule directory supported to be here?

Reply:
ah looks like it's for the tests....... It can be moved to a /tests/fixtures/molecule directory

Reply:
so should I move it or not in this PR?